pyproject.toml

```diff
@@ -1,3 +1,94 @@
+[build-system]
+requires = ["flit_core >=3.2,<4"]
+build-backend = "flit_core.buildapi"
+
+[tool.flit.sdist]
+include = [
+    "AUTHORS.rst",
+    "CITATION.cff",
+    "CODE_OF_CONDUCT.md",
+    "CONTRIBUTING.rst",
+    "LICENSE",
+    "README.rst",
+    "VERSION",
+    "tox.ini",
+    "ci/",
+    "docs/",
+    "examples/",
+    "src/",
+    "tests/",
+    ".bumpversion.cfg",
+    ".coveragerc",
+    ".editorconfig",
+    ".flake8",
+    ".pep8speaks.yml",
+    ".readthedocs.yaml",
+    ".scrutinizer.yml",
+]
+exclude = ["docs/_build"]
+
+[project]
+name = "oemof.solph"
+dynamic = ["version"]
```

Comment:
This reads the version number from the `__version__` variable of the project, one could also make the `description` dynamic, that would be the module docstring in the projects top level `__init__.py`.